src/VirtoCommerce.NotificationsModule.Data/Senders/NotificationSender.cs

```diff
@@ -31,6 +32,12 @@ public class NotificationSender : INotificationSender
 
         public void ScheduleSendNotification(Notification notification)
         {
+            //need to except 'is-readonly' notifications because after serialization generated couple 'is-readonly' templates
```

Comment:
How the `hardcoded` notifications will work in this case?

Reply:
yes, I mean 'is-readonly' is hardcoded notifications

Reply:
we decided need to add unit test for it if it is possible.